index.html

```diff
@@ -1,9 +1,24 @@
-<html>
-    <head>
-        <title>McGregor FTProject</title>
-    </head>
-    <body>
-        <h1>Welcome to my cool website!</h1>
-        <script src="dist/app.bundle.js" async></script>
-    </body>
+<!doctype html>
+<html lang="en">
+<head>
+    <title>McGregor FTProject</title>
+
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Include Lato Font -->
+    <link href="https://fonts.googleapis.com/css?family=Lato:300,400|Material+Icons" rel="stylesheet">
+
+    <!-- Include our global stylesheet -->
+    <link href="dist/app.css" rel="stylesheet">
+</head>
+<body>
+
+    <!-- We're utilizing RactiveJS to create and manage this DOM. See ./src/app.js for details. -->
+    <section id="BrandonsFTProject"></section>
```

Comment:
Utilizing [Ractive](https://ractive.js.org/), we render and two-way bind our [view-data](https://github.com/IAmBrandonMcGregor/mcgregor-ftproject/pull/1/files#diff-bd9c9dcd314f2d7df52935b3a6a4d504R25) to our [app.html](https://github.com/IAmBrandonMcGregor/mcgregor-ftproject/pull/1/files#diff-b215bf4a096825a90f6466179156d243) template.